iot-e2e-tests/common/src/main/java/com/microsoft/azure/sdk/iot/common/tests/iothubservices/twin/QueryTwinTests.java

```diff
@@ -40,6 +42,7 @@ public QueryTwinTests(String deviceId, String moduleId, IotHubClientProtocol pro
     }
 
     @Test
+    @ConditionalIgnoreRule.ConditionalIgnore(condition = StandardTierOnlyRule.class)
```

Comment:
>ConditionalIgnoreRule [](start = 5, length = 21)

can this be applied at class level?

Reply:
not class level

---
In reply to: [274581158](https://github.com/Azure/azure-iot-sdk-java/pull/485#discussion_r274581158) [](ancestors = 274581158)